examples/model_selection/plot_train_error_vs_test_error.py

```diff
@@ -86,3 +86,45 @@
 plt.legend()
 plt.subplots_adjust(0.09, 0.04, 0.94, 0.94, 0.26, 0.26)
 plt.show()
+
+# %%
+# Plotting Validation Curves
```

Comment:
Please amend the plot above because the top figure correspond to the validation curve. So you should only use the display.

You might need to add the vline on the display but it will be quite straightforward.